MyPerfectOnboardingApplication/Tests/MyPerfectOnboarding.Tests.Utils/Extensions/ArgExtended.cs

```diff
@@ -0,0 +1,12 @@
+﻿using MyPerfectOnboarding.Contracts.Models;
+using MyPerfectOnboarding.Tests.Utils.Comparers;
+using NSubstitute;
+
+namespace MyPerfectOnboarding.Tests.Utils.Extensions
+{
+    public static class ArgExtended
+    {
+        public static ListItem IsListItem(IListItem expectedItem)
+            => Arg.Is<ListItem>(item => ListItemEqualityComparer.Instance.Equals(item, expectedItem));
```

Comment:
```suggestion
            => Arg.Is<ListItem>(acutalItem => ListItemEqualityComparer.Instance.Equals(acutalItem, expectedItem));
```
This line, standing alone, almost suggests the name`actualItem` all on its own 🤣